subjects/strisnegative/README.md

```diff
@@ -0,0 +1,43 @@
+## isnegative
+
+Write a function named `IsNegative()` that defines if a number (You should check if it's a number) is negative or positive.
```

Comment:
Write a function called `IsNegative()` that defines whether a number is negative or positive.
- Your function prints `P` if the number is positive
- Your function prints `F` if the number is negative
- If the number is zero, print `0`.
- If it's not a number, print `('\n') `.
- Your program should always print `('\n') ` at the end of the output.